packages/bot-web-ui/src/components/self-exclusion.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input, ThemedScrollbars, Button } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { Formik, Form, Field } from 'formik';
+import { connect } from '../stores/connect';
+
+class SelfExclusion extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            self_exclusion_hide: true,
+            limit_error: '',
+        };
+    }
+    componentDidMount() {
+        if (!this.props.max_losses) {
+            this.setState({ self_exclusion_hide: false });
+        }
+    }
+    render() {
+        const {
+            max_losses,
+            set_max_losses,
+            initial_values,
+            updateSelfExclusion,
+            setLimitations,
+            onRunButtonClick,
+            run_limit,
+        } = this.props;
+        const self_exclusion_status = max_losses || 'Not set';
+
+        const onCancelSelfExclusion = async () => {
+            const { switchAccount, virtual_account_loginid, checkRestrictions } = this.props;
+            await switchAccount(virtual_account_loginid);
```

Comment:
There is no need to switch account when the user cancels the self-exclusion dialog. Next time they try to run the bot, we'll just prompt them again.

Reply:
> There is no need to switch account when the user cancels the self-exclusion dialog. Next time they try to run the bot, we'll just prompt them again.

Actually it is not canceling self-exclusion button it is the button which we have to switch to the demo account 
to solve this problem I change the name of function

Reply:
> > There is no need to switch account when the user cancels the self-exclusion dialog. Next time they try to run the bot, we'll just prompt them again.
> 
> Actually it is not canceling self-exclusion button it is the button which we have to switch to the demo account
> to solve this problem I change the name of function

Update to onSwitchAccount